### DISCUSSION TITLE
(投票) 是否解放仅限虚拟主播及闪电主播的记录限制

### DISCUSSION THREAD

**Discussion by eric2788** [eric2788/ddstats.ericlamm.xyz#3] [Polls]
目前基于隐私问题限制了非虚拟主播和闪电主播的记录限制。现正考虑是否应该解放限制让记录行为扩大到普通B站用户。

通过后，直播间监控将基于目前仅限虚拟主播及闪电主播作为记录，但行为记录则会概括全体B站用户，并且进行统计化排行及分析。

**Comment by SakuraChiyodaisuki** [comment]
danmaku才吃睿弹，为啥现在提这个

**Reply by eric2788** [reply]
別急，danmaku 快復活了
而且吃睿弹是因為流水統計，而不是查成分

**Reply by lavie-veux** [reply]
流水統計好有用，内研写一半吃睿弹了，发愁

**Reply by SakuraChiyodaisuki** [reply]
> 別急，danmaku 快復活了 而且吃睿弹是因為流水統計，而不是查成分

流水统计吃睿弹可还行，danmaku的流水都是用的matsuri的数据，对比darkflame和vtbfun都差了不少，因为这个吃睿弹多少有些冤枉

**Reply by eric2788** [reply]
> > 別急，danmaku 快復活了 而且吃睿弹是因為流水統計，而不是查成分
> 
> 流水统计吃睿弹可还行，danmaku的流水都是用的matsuri的数据，对比darkflame和vtbfun都差了不少，因为这个吃睿弹多少有些冤枉

首先，matsuri的流水统计比danmaku更不显眼，所以很多人截图引用的基本上很少见matsuri, 反而是 darkflame / danmaku 居多；其次也是最主要原因还是有人在公会面前发了danmaku流水统计的截图才导致此事的

**Comment by WhiteKnight300** [comment]
这块功能预计近期会上线吗？

**Reply by eric2788** [reply]
否，最近暂时没什么空

**Comment by eric2788** [comment]
不错，最近开始记录了普通用户的记录，结果没过几天储存空间已经超几G了

**Comment by eric2788** [comment]
功能已上線
